src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4447,7 +4450,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
 
                         - CODE GENERATION -
  /debug[+|-]                   Emit debugging information
- /debug:{full|pdbonly|portable|embedded}
+ /debug:{{full|pdbonly|portable|embedded}}
```

Comment:
I'm now using this string as a template for formatting. The curly braces are part of interpolated string format. Double curly braces are the way to escape single curly braces.